tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckForExemplars.java

```diff
@@ -60,9 +62,18 @@ public class CheckForExemplars extends FactoryCheckCLDR {
 
     private static final String STAND_IN = "#";
 
-    // private final UnicodeSet commonAndInherited = new
-    // UnicodeSet(CheckExemplars.Allowed).complement();
-    // "[[:script=common:][:script=inherited:][:alphabetic=false:]]");
+    /**
+     * These values, and their uppercase variants, are forbidden for any path. They should all be
+     * lowercase.
+     */
+    private static final List<String> FORBIDDEN_VALUES = new ArrayList<>(Arrays.asList("n/a"));
```

Comment:
OK for now, I can see this eventually needing to  be a regex but that can be future.